rules/prefer-global-this.js

```diff
@@ -0,0 +1,259 @@
+'use strict';
+
+const {GlobalReferenceTracker} = require('./utils/global-reference-tracker.js');
+
+const MESSAGE_ID_ERROR = 'prefer-global-this/error';
+const MESSAGE_ID_SUGGESTION = 'prefer-global-this/suggestion';
+const messages = {
+	[MESSAGE_ID_ERROR]: 'Prefer `{{replacement}}` over `{{value}}`.',
```

Comment:
The `replacement` is always `globalThis`, no need use template.

Reply:
resolved. 461014aefa976b59a6b8c86397248f711600f415